cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -1,4 +1,113 @@
-Usage: cardano-cli (Era based commands | byron | ping | version)
+Usage: cardano-cli 
+                     ( shelley
+                     | allegra
+                     | mary
+                     | alonzo
+                     | babbage
+                     | conway
+                     | latest
+                     | experimental
```

Comment:
These are the new command groups.

Note that `byron` is intentionally missing because there already is a legacy `byron` command group.

`latest` currently set to `byron` and `experimental` set to `conway`.